src/blocks/checkout/shipping.tsx

```
@@ -0,0 +1,302 @@
+  } = useCheckoutStore();
+
+  const [selectedCountry, setSelectedCountry] = useState(
+    shippingFormData.country || countries[0]?.cca2 || ''
+  );
+  const [disabledFields, setDisabledFields] = useState<boolean>(false);
+
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    watch,
```

ESLint

[error] 'watch' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)